app/assets/stylesheets/exception_hunter/base.css

```diff
@@ -71,3 +71,12 @@ form {
 .button.button-dismiss:hover, .button.button-dismiss:focus, .button.button-dismiss:active {
     color: var(--inactive-grey);
 }
+
+.column-8{
+    flex: 0 0 8% !important;
+    max-width: 8% !important;
```

Comment:
can we have this ones to be more specific instead of having the `!important` ?  😬